storagemanager/storage/tests.py

```diff
@@ -1,3 +1,16 @@
 from django.test import TestCase
+from .models import Item
+from . import views
 
-# Create your tests here.
+
+class  ItemTestCase(TestCase):
```

Comment:
Nice! Good to have first tests. In future you can have a look at `pytest` library that is popular for python testing.